### ISSUE TITLE
[Suggestions] Few enhancements proposal

### ISSUE THREAD

**Issue by Ouack23** [librehat/yapstocks#8]
Hello,
First let me tell you that I really like your widget.

Here are a few suggestions that would be nice to have (while not being supa dupa important) : 
* Include pre and post market prices
* Allow the user to configure the app to send desktop notifications when stock goes above and / or below certain limits
* <5 minutes update intervals

What do you think about this ?
Thanks for reading !

**Comment by librehat** [comment]
Thank you!

I've had some ideas about a more feature-rich application for stocks monitoring (or positions monitoring). However, I personally see that more useful as a standalone application rather than a desktop widget. It will take some time to make a standalone application but that's more flexible than the form of a Plasma widget. What do you think?

**Comment by Ouack23** [comment]
That would be cool indeed ! The few apps that I tried weren't that good or simply didn't work at all.
I could make a PKGBUILD for your future app in the archlinux AUR repo :)
I am currently doing a cross platform Qt application so I could be of some help if you plan to use Qt. If not, maybe I could still help a little ...
Thanks for your quick answer.

I edited my first post to include faster stock value update.
